### PR TITLE
pass options to transforms as `opts._flag`

### PR DESCRIPTION
**Pull request by ahdinosaur**
fixes https://github.com/substack/node-browserify/issues/1463
supersedes https://github.com/substack/module-deps/pull/61

### CODE REVIEW

File: test/tr_flags.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            options: {}
+        })
+        p.on('error', function (err) { return t.fail(err.message) })
+        p.end(empty);
+    });
+
+    p = mdeps({ debug: true })
+    p.write({
+        transform: function (file, opts) {
+            t.equal(opts._flags, Infinity, 'transform arguents are preserved');
```
Comment (yoshuawuyts) [inline test/tr_flags.js]: `s/arguents/arguments/`

**Comment by yoshuawuyts** [comment]
:+1: looks useful!

**Comment by ahdinosaur** [comment]
hey @zertosh @substack, any chance this can be merged and published? would be much appreciated. :cake:

**Comment by yoshuawuyts** [comment]
ping @terinjokes any chance this could land?

**Pull request merged**